web/src/containers/Expenses/index.js

```diff
@@ -0,0 +1,59 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Switch, Route } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { push } from 'react-router-redux';
+import { Box } from 'rebass';
+import { bindActionCreators } from 'redux';
+
+import FormLogger from '../../util/formLogger';
+
+import CommonExpenses from './Common';
+import CategoryList from './List';
+import EditExpense from './Edit';
+
+class ExpensesLanding extends Component {
+  showResults = () => {};
+
+  render() {
+    const { goTo } = this.props;
+    const root = this.props.match.path;
+
+    return (
+      <Box py={4}>
+        <FormLogger />
+        <Switch>
+          <Route
+            path={`${root}/list`}
+            component={() => (
+              <CategoryList goTo={goTo} next="/apd-overview" prev={root} />
+            )}
+          />
+          <Route
+            path={`${root}/edit`}
+            component={() => <EditExpense goTo={goTo} next={`${root}/list`} />}
```

Comment:
so to pass in props to a component for `Route` you have to wrap in the function? not the prettiest but I guess it works!

Reply:
Yeah, we can definitely find a better solution.  This was just get-it-out-the-door code.